app/modules/elasticsearch/tasks.py

```diff
@@ -128,7 +128,43 @@ def load_encounters_index():
 """
 
 
+def load_sightings_index():
+    wb_engine = create_wildbook_engine()
+    with wb_engine.connect() as wb_conn:
+        results = wb_conn.execute(text(SIGHTINGS_INDEX_SQL))
+        for result in results:
+            # Create the document object
+            sighting = Sighting(**result)
+            # Assign the elasticsearch document identify
+            sighting.meta.id = f'sighting_{sighting.id}'
+            # Save document to elasticsearch
+            sighting.save(using=current_app.elasticsearch)
+
+
+SIGHTINGS_INDEX_SQL = """\
+SELECT
+  oc."ID" AS id,
+  NULLIF((oc."DECIMALLATITUDE"::float || ',' || oc."DECIMALLONGITUDE")::text, ',') AS point,
+  (
+    SELECT max("GENUS")
+    FROM "ENCOUNTER" en
+      RIGHT JOIN "OCCURRENCE_ENCOUNTERS" oe ON oe."ID_EID" = en."ID"
+    WHERE oe."ID_OID" = oc."ID") AS genus,
+  (
+    SELECT max("SPECIES")
+    FROM "ANNOTATION" an
```

Comment:
annotations have been moved to houston, so these are historical (migration) use only.   the Encounter-Annotation join is now in houston.  (as is anything we might eventually want to see from Annotation in elasticsearch, like keywords.  but this can wait til a later PR, imho.)